metaci/users/management/commands/verify_environment.py

```diff
@@ -0,0 +1,105 @@
+import os
+
+from cumulusci.core.github import get_github_api_for_repo
+from cumulusci.oauth.salesforce import CaptureSalesforceOAuth
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from requests.adapters import HTTPAdapter
+
+from metaci.build.tasks import scratch_org_limits
+from metaci.cumulusci.keychain import GitHubSettingsKeychain
+from metaci.cumulusci.utils import get_connected_app
+
+
+class Command(BaseCommand):
+    help = "Test the installation environment."
+
+    def handle(self, *args, **options):
+        successes = []
+        errors = []
+
+        tests = [
+            (
+                scratch_org_limits,
+                "Devhub seems to be configured",
+                "Devhub is not configured. Scratch org creation will fail",
+            ),
+            (
+                test_github,
+                "Github API is configured",
+                "Github API is not configured. Repos cannot be retrieved for builds.",
+            ),
+            (
+                test_connected_app,
+                "Connected app environment variables allowed oauth",
+                "Connected app is not configured.  Persistent org connection will fail",
+            ),
+            (
+                test_github_webhook_setting,
+                "Github Webhook Secret is set, but cannot be tested",
+                "Github Webhook Secret is not set. Github actions will not be able to trigger builds.",
+            ),
+        ]
+
+        for test in tests:
+            func, good, bad = test
+            try:
+                func()
+                successes.append(good)
+            except BaseException as e:
+                errors.append(bad + f": {e}")
+
+        print("")
+        print("")
+
+        for success in successes:
+            print("SUCCESS:", success)
+
+        if errors:
+            print()
+            for error in errors:
+                print("ERROR:", error)
+            print()
+            print("There were errors:")
+            print("Check your .env file or environment.")
+            print("Please read configuring.rst for more details.")
+
+
+def test_connected_app():
+    # is there a way to test this with no user interaction?
```

Comment:
Not without the instance_url and username of a user that has already approved the connected app